CMakeLists.txt

```diff
@@ -2,19 +2,11 @@ cmake_minimum_required(VERSION 3.13.4)
 
 project("SVF")
 
+message(STATUS "Using CMake with generator: ${CMAKE_GENERATOR}")
+
 configure_file(${PROJECT_SOURCE_DIR}/.config.in
                ${PROJECT_BINARY_DIR}/include/Util/config.h)
 
-# We need to match the build environment for LLVM: In particular, we need C++14
-# and the -fno-rtti flag
-set(CMAKE_CXX_STANDARD 14)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-# add -std=gnu++14
-set(CMAKE_CXX_EXTENSIONS ON)
-
-add_compile_options("-fno-rtti")
-add_compile_options("-fno-exceptions")
```

Comment:
If we want to enable svfcore as `-fno-rtti/fno-exceptions`, these two options should also be in the root/CMakeList or svf/CMakeList?

Reply:
Sure, this was why I initially moved the LLVM finding into the top-level `CMakeLists.txt` since settings these options is based on whether the found LLVM instance supports them. I changed it now so that the `CMakeLists.txt` in `svf-llvm` sets a variable in its parent scope, which is then checked in the top-level `CMakeLists.txt` to conditionally set the flags for all targets.